extensions/amp-ad/0.1/amp-ad-3p-impl.js

```diff
@@ -233,10 +233,11 @@ export class AmpAd3PImpl extends AMP.BaseElement {
       // incrementLoadingAds().
       this.lifecycleReporter.sendPing('adRequestStart');
       const iframe = getIframe(this.element.ownerDocument.defaultView,
-          this.element, undefined, opt_context);
+                               this.element, undefined, opt_context);
+      const sentinel = JSON.parse(iframe.name).attributes._context.sentinel;
```

Comment:
This is not a JSON encoded value, it's https://github.com/ampproject/amphtml/pull/6506/files#diff-34e84f9cf5e8134d86bf1fc893ab293aR123